slight reword and detail added (#38)
Faithful-Resource-Pack/Docs: pages/manuals/add-on-rules.md

---
title: "Rules for Add-ons"
category: "Manuals"

> date: "2022-08-13"

Reviewer: this is for the initial publish date (unless basically the entire doc was rewritten), not for updates

---

# Rules for Faithful Add-ons

> 1. **Your pack must be based on the latest Faithful textures at the time of publishing.** You can check on the [Web App](https://webapp.faithfulpack.net/gallery/java/faithful_32x/java-latest/all) for the latest textures contributed.

Reviewer: ```suggestion
1. **Your pack must be based on the latest Faithful textures at the time of publishing.** You can check the [web app](https://webapp.faithfulpack.net/gallery/java/default/latest/all) for the latest contributed textures.
```

2. **All images must accurately represent your add-on in its current state.** Don’t use any packs other than Faithful with your add-on applied while taking screenshots!

> 3. **If at all applicable, all textures in your add-on must at least partially adhere to the texturing guidelines.** 

Reviewer: why remove this, it still applies

Author: Ehhh who's really making addons for CF64

Reviewer: you can't ignore entire packs just because it isn't as active

3. **If at all applicable, all textures in your add-on must at least partially adhere to the texturing guidelines.** 
4. **The title of your add-on must comply with these rules:**
    <ol class="roman">
    <li><b>It must be descriptive enough to give anybody a basic idea what your pack does.</b></li>
    <li><b>It must be descriptive enough to give anybody a basic idea of what your pack does.</b></li>
    <li><b>It has to be properly capitalised with <a href="https://en.wikipedia.org/wiki/Title_case">Title Case</a>.</b></li>
    <li><b>When possible, use proper English grammar.</b>
    <br>If you forget to do any of these nothing bad will happen, but please keep this in mind so that the managers don’t have to fix the title for every add-on.</li>
    <li><b>Avoid using the word “Better” in the name!</b>
    <br>Be creative and describe your pack more precisely. This can be tolerated in some cases if the header image shows what the add-on does clearly, but we'd still prefer more creative names for your add-ons.</li>
    <li><b>No obscenities or swear words.</b></li>
    <li><b>Don’t use the word “Faithful” in the title unless absolutely necessary.</b>

>     <br>It’s already evident that your pack is made for Faithful, since, you know, it’s on the Faithful website. The same goes for the resolution—don't include it in your title, since it's displayed right above it anyway.</li>

Reviewer: I like the added justification, I feel like that should stay

Author: The docs are already childish enough let it go already, I'll probably reword at little bit more if that's what you want

# Rules for Faithful Add-ons

> 1. **Your pack must be based on the latest Faithful textures at the time of publishing.** You can check the [web app](https://webapp.faithfulpack.net/gallery/java/default/latest/all) for the latest contributed textures.

Reviewer: I'd say texture gallery instead of web app here honestly, the webappiness isn't as relevant as the gallery

Author: That was your change but mkay

2. **All images must accurately represent your add-on in its current state.** Don’t use any packs other than Faithful with your add-on applied while taking screenshots!

> 3. **If at all applicable, all textures in your add-on must at least partially adhere to the texturing guidelines.** This only applies to certain packs that have guidelines.

Reviewer: ```suggestion
3. **If at all applicable, all textures in your add-on must at least partially adhere to the texturing guidelines.** This only applies to packs with guidelines.
```

4. **The title of your add-on must comply with these rules:**
    <ol class="roman">
    <li><b>It must be descriptive enough to give anybody a basic idea of what your pack does.</b></li>
    <li><b>It has to be properly capitalised with <a href="https://en.wikipedia.org/wiki/Title_case">Title Case</a>.</b></li>
    <li><b>When possible, use proper English grammar.</b>

>     <br>If you forget to do any of these above, nothing bad will happen, but please keep this in mind so that the managers don’t have to fix the title for every add-on.</li>

Reviewer: this makes no sense grammatically (if you really want to include above you should say "If you forget to do any of the above things" or something like that) but honestly I feel like it's pointless when it's also nested and pretty obviously grouped

Author: tbf this sounds like it should be below 4. as its own point but I don't want to majorly rewrite it

    <li><b>Avoid using the word “Better” in the name!</b>
    <br>Be creative and describe your pack more precisely. This can be tolerated in some cases if the header image shows what the add-on does clearly, but we'd still prefer more creative names for your add-ons.</li>
    <li><b>No obscenities or swear words.</b></li>
    <li><b>Don’t use the word “Faithful” in the title unless absolutely necessary.</b>

>     <br>It’s already evident that your pack is made for Faithful, since you are downloading it from the Faithful website. The same goes for the resolution — don't include it in your title, since it's already displayed right above it.</li>

Reviewer: never add spaces around an em dash (it's _technically_ not wrong but nothing else does this and I find it really ugly)

>     <br>It’s already evident that your pack is made for Faithful, since you are downloading it from the Faithful website. The same goes for the resolution—don't include it in your title, since it's already displayed right above it.</li>

Author: more like its right next to the title :shrug:
